github/dependabot.md

```diff
@@ -24,30 +24,55 @@ For instance, [here are some vulnerabilities found in the k8s.io libraries we us
 
 ## Configuration
 
-There are two ways to configure Dependabot:
+Dependabot updates can be enabled for a repository in the "Settings & analysis" tab of the GitHub settings UI.
+To enable updates, simply click the "Enable" button as shown in the image below.
+![](dependabot/dependabot-enable.png)
 
-1. Through [the web UI](https://app.dependabot.com/accounts/giantswarm/)
-2. Using [configuration files](https://dependabot.com/docs/config-file/) (preferred)
+To further configure how Dependabot performs updates in your repository, use a configuration file.
+You can generate a stub configuration file using `devctl gen dependabot` (see [`devctl`](https://github.com/giantswarm/devctl) for usage).
```

Comment:
Is there a reason to maintain this file manually when we generate it with `devctl`?

@njuettner is it supposed to be a stub when the file is regenerable in your implementation?

Reply:
Ideally everything should be handled via devctl. 
Right now I only focussed on implementing it for go projects, I'm not aware how many other repos we have which use different languages.

Reply:
Okay. @stone-z can we reflect that here? I mean that for **all** Go projects we should use what `devctl gen dependabot` generates.

Reply:
We have projects which are not Go, and we have different needs between Go projects, so IMO yes

Reply:
Sorry, previous comment was based on old GH cached state. Updated